src/apps/Agreement/prop-types.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types'
+
+const AppItemType = PropTypes.shape({
```

Comment:
Let's alphabetize :).

I'm a bit of a stickler for this as a convention, but I do think it makes it easier to scan for something when looking back. The only times we don't stick to this convention is when there is a strong case to be made about priority (e.g. either there's a very obvious order / importance, such as importing React first, or very obvious grouping, such as `startDate` and `endDate`).